### PR TITLE
zh: add TLS for mysql client document

### PR DESCRIPTION
**Pull request by weekface**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/enable-tls-for-mysql-client.md [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+创建一套 Client 端证书。可以使用官网提供的方式创建证书或者使用 `cfssl`
+创建证书。
+
+#### 官网创建证书
+
+请参考官网文档：[Enable TLS for MySQL Clients | TiDB Documentation](https://pingcap.com/docs/stable/how-to/secure/enable-tls-clients/)。
+
+#### 使用 cfssl 创建证书
+
+##### 下载 cfssl 软件并初始化证书颁发机构
```
Comment (ran-huang) [inline zh/enable-tls-for-mysql-client.md]: 一般不建议使用超过四级的标题，可以把这里和下面的五级标题都改成列表，注明第 `n` 步，方便用户按顺序操作

File: zh/enable-tls-for-mysql-client.md [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+
+mkdir -p ~/cfssl
+cd ~/cfssl
+cfssl print-defaults config > ca-config.json
+cfssl print-defaults csr > ca-csr.json
+```
+
+##### 配置 CA 选项
+
+{{< copyable "shell-regular" >}}
```
Comment (ran-huang) [inline zh/enable-tls-for-mysql-client.md]: The code block with `copyable` snippet should be the command that users need to copy and past in operation.
Please confirm that the following code block conforms to this rule.

File: zh/enable-tls-for-mysql-client.md [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+$ cfssl gencert -initca ca-csr.json | cfssljson -bare ca -
+
+ca-key.pem
+ca.csr
+ca.pem
+```
+
+##### 生成 Server 端证书
+
+首先生成默认的 `server.json` 文件
```
Comment (ran-huang) [inline zh/enable-tls-for-mysql-client.md]:
```suggestion
首先生成默认的 `server.json` 文件：
```

File: zh/enable-tls-for-mysql-client.md [hunk 4]
```diff
@@ -0,0 +1,464 @@
...
+$ cfssl gencert -ca=ca.pem -ca-key=ca-key.pem -config=ca-config.json -profile=server server.json | cfssljson -bare server
+
+server-key.pem
+server.csr
+server.pem
+```
+
+##### 生成 Client 端证书
+
+首先生成默认的 `client.json` 文件
```
Comment (ran-huang) [inline zh/enable-tls-for-mysql-client.md]:
```suggestion
首先生成默认的 `client.json` 文件：
```

File: zh/enable-tls-for-mysql-client.md [hunk 5]
```diff
@@ -0,0 +1,464 @@
...
+- [创建 TiDB 集群](#创建-TiDB-集群)
+- [MySQL 客户端使用方法](#MySQL-客户端使用方法)
+
+## 概述
+
+TiDB Operator 从 v1.1 开始已经支持为 Kubernetes 上的 TiDB 集群开启 MySQL 客户端 TLS。
+
+开启此功能的步骤为：
+
+- 为即将创建的 TiDB 集群生成两套证书：为 TiDB Server 创建一套 Server
```
Comment (ran-huang) [inline zh/enable-tls-for-mysql-client.md]: Please don't write one sentence in multiple lines.

File: zh/enable-tls-for-mysql-client.md [hunk 6]
```diff
@@ -0,0 +1,464 @@
...
+ cluster:
+   namespace: default
+   name: basic
+ initSql: |-
+   create database app;
+```
+
+上面通过设置 `spec.tidb.tlsClient.enabled` 属性为 `true` 来开启 MySQL
+客户端 TLS：
+
```
Comment (ran-huang) [inline zh/enable-tls-for-mysql-client.md]: Remove the extra blank line
```suggestion
```

**Comment by weekface** [comment]
@ran-huang 不知道为什么合并了 master 以后，有很多冲突，很艰难解决掉这些冲突。重新提了个 PR：https://github.com/pingcap/docs-tidb-operator/pull/26

已经按照 comments 都做了修改。麻烦你再看下，多谢🙏。

**Pull request closed without merging**